api/applications/applications_controller.go

```diff
@@ -388,7 +394,17 @@ func (ac *applicationController) RegenerateMachineUserTokenHandler(accounts mode
 
 	appName := mux.Vars(r)["appName"]
 	handler := ac.applicationHandlerFactory(accounts)
-	machineUser, err := handler.RegenerateMachineUserToken(appName)
```

Comment:
There is logic to parse daysUntilExpiry from web console. It defaults to 365 days. This value is used to compute an expiration timestamp annotation on the Secret objects. However, this expiration timestamp has no effect (yet).